main.go

```diff
@@ -537,17 +537,17 @@ func main() {
 				Destination: &org,
 			},
 		},
-		Action: func(cCtx *cli.Context) error {
-			db, err := connectDB(org)
+		Before: func(cCtx *cli.Context) error {
+			var err error
+			db, err = connectDB(org)
```

Comment:
Shouldn't we get `org` from a flag? Looks like it's always `default` for now?

Reply:
Line 537 has the code for getting org from a flag, the 'Destination:' part sends the value from the flag to the org variable.